README.md

```diff
@@ -0,0 +1,48 @@
+#### Table of Contents
```

Comment:
This file's already here as README.markdown, can you remove the old one? (Git/GitHub should automatically see the rename.)